data/ga/legislature/Colton-Moore-33b353c3-50e9-485f-b154-0d5310da4f2f.yml

```diff
@@ -0,0 +1,28 @@
+id: ocd-person/33b353c3-50e9-485f-b154-0d5310da4f2f
+name: Colton Moore
+given_name: Colton
+family_name: Moore
+email: colton.moore@senate.ga.gov
+party:
+- name: Republican
+roles:
```

Comment:
@sroomf this person served in the Georgia House District 1 from Jan 2018 until Jan 2020. So I think they should already have a file but it wouldn't be in this pull? @bfossen-ce solved some of these when we were in Chicago.

Reply:
there was one version of his file that was retired and one version in legislature! i think it's a linting error i didn't catch on my first pass, thanks for noticing it!